### ISSUE TITLE
Normative and Informative references (Trac #35)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#35]
References are now required to be split into "Normative" and "Informative".

Migrated from https://trac.ietf.org/ticket/35
```json
{
    "status": "closed",
    "changetime": "2008-01-05T12:30:13",
    "_ts": "1199536213000000",
    "description": "References are now required to be split into \"Normative\" and \"Informative\".",
    "reporter": "mnot@pobox.com",
    "cc": "",
    "resolution": "fixed",
    "time": "2007-12-20T05:07:46",
    "component": "non-specific",
    "summary": "Normative and Informative references",
    "priority": "",
    "keywords": "",
    "milestone": "01",
    "owner": "fielding@gbiv.com",
    "type": "editorial",
    "severity": ""
}
```

**Comment by mnot** [comment]
Trac update at 20080101T15:02:49: 
* julian.reschke@gmx.de changed component from "" to "non-specific"
* julian.reschke@gmx.de changed owner from "" to "julian.reschke@gmx.de"

**Comment by mnot** [comment]
Trac update at 20080101T15:16:27: julian.reschke@gmx.de commented:

> Note that there are several other issues related to this: #44, #45, #46, #55, #65, #66, #68, #86

**Comment by mnot** [comment]
Trac update at 20080101T15:19:52: julian.reschke@gmx.de commented:

> Some changes in [132]
>

**Comment by mnot** [comment]
Trac update at 20080104T06:15:57: mnot@pobox.com changed milestone from "" to "unassigned"

**Comment by mnot** [comment]
Trac update at 20080105T01:38:13: 
* fielding@gbiv.com changed owner from "julian.reschke@gmx.de" to "fielding@gbiv.com"
* fielding@gbiv.com changed status from "new" to "assigned"
* fielding@gbiv.com changed version from "" to "00-draft"

**Comment by mnot** [comment]
Trac update at 20080105T03:47:40: fielding@gbiv.com changed milestone from "unassigned" to "01"

**Comment by mnot** [comment]
Trac update at 20080105T03:48:15: fielding@gbiv.com changed type from "design" to "editorial"

**Comment by mnot** [comment]
Trac update at 20080105T11:54:39: julian.reschke@gmx.de commented:

> From [151]:
> 
> Categorize reference to RFC1737 as informative (addresses #35)
>

**Comment by mnot** [comment]
Trac update at 20080105T12:30:13: 
* julian.reschke@gmx.de commented:

> Fixed in [152]:
> 
> Resolve #35: categorize RFC1737 (URN) as informative, RFC4288 (media type registration procedure) as normative (closes #35). Also update and re-organize media type registration for application/http and message/http to use RFC4288 template (relates to #8).
> 


* julian.reschke@gmx.de changed resolution from "" to "fixed"
* julian.reschke@gmx.de changed status from "assigned" to "closed"
